README.md

```diff
@@ -1,3978 +1,7 @@
-# Kottans-frontend
```

Comment:
You deleted 3979 lines in README.md - kind of decided to leave only your comment there? :rofl:

Reply:
i`m so sorry😭